### PR TITLE
Wiz docs

### PR DESCRIPTION
**Pull request by Wizzerrd**
Changes:
- External links open in a new tab
- Changed verbiage on the splash page
- Changed featured cards on splash page
- Created Introduction Page for Documentation
- Added segments to the References section
- Added sections with segments to Getting Started:
  - Quickstart 
  - Demos
  - Tutorials

Notes:
General
- I will build and deploy an agent this week from the ground up using the Eidolon framework. This will give me better context to - help answer some of the following questions and resolve the following issues.
- What is Eidolon? Justify Existence. Add to Docs Under new section
  - What is an Agent?
  - Why do you need an Agent Services Framework?

- Elaborate on Why Eidolon Sections / Provide Examples
  - Easy to Deploy
  - Simple Agent-to-Agent Communication
  - Painless Component Customization and Upgrade

Introduction:
- Context behind Eidolon
  - What are Agents
  - What is an Agent Services Framework
- What is Eidolon
- Why use Eidolon - Elaborate

Quickstart Guide:
- Create an Agent needs content
- All pages need elaboration

Demos:
- More Demos

Tutorials:
- I will walk through tutorials and assess usability this week

References:
- I will walk through tutorials and assess usability this week
- Type/class definitions?

Architecture:
- I will walk through tutorials and assess usability this week
- Type/class definitions?

### CODE REVIEW

File: docs/src/content/docs/references/communication.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+_hello_world_agent.yaml_
+```yaml
+apiVersion: eidolon/v1
+kind: Agent
+metadata:
+  name: hello_world
+
+spec:
+  implementation: GenericAgent
```
Comment (LukeLalor) [inline docs/src/content/docs/references/communication.md:16]: We want to deprecate the GenericAgent. Can you move this example to the "SimpleAgent" (used by default) or make a separate issue to make the changes?

**Review by LukeLalor — APPROVED** [review]
left some comments about improving existing docs. I assume you just copied it over from the quickstart, so feel free to break that into a different issue if you would like.

File: docs/src/content/docs/references/pluggable.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+_frugal_cpu.yaml_
+```yaml
+apiVersion: eidolon/v1
+kind: Reference
+metadata:
+  name: frugal_cpu
+
+spec:
+  implementation: "eidolon_ai_sdk.cpu.conversational_agent_cpu.ConversationalAgentCPU"
```
[the remark this answers is not in the document]
Comment (Wizzerrd) [reply docs/src/content/docs/references/pluggable.md]: Fixed with commit [5659372](https://github.com/eidolon-ai/eidolon/pull/146/commits/565937205109d552cd7372d78ccc33878e18a036)

File: docs/src/content/docs/references/defining.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+_machine.yaml_
+```yaml
+apiVersion: eidolon/v1
+kind: Machine
+metadata:
+  name: mongo_machine
+
+spec:
+  symbolic_memory:
+    implementation: "eidolon_ai_sdk.memory.mongo_symbolic_memory.MongoSymbolicMemory"
```
[the remark this answers is not in the document]
Comment (Wizzerrd) [reply docs/src/content/docs/references/defining.md]: Fixed with commit[7af8af2](https://github.com/eidolon-ai/eidolon/pull/146/commits/7af8af25013c0f2b3ad9128391e05bd799419990)

File: docs/src/content/docs/references/communication.md (hunk 1 above)
Comment (Wizzerrd) [reply docs/src/content/docs/references/communication.md:16]: Issue opened: [#174](https://github.com/eidolon-ai/eidolon/issues/174)

**Pull request merged**